swift/BUILD.bazel

```diff
@@ -77,12 +77,12 @@ pkg_filegroup(
         ":extractor",
         ":swift-test-sdk-arch",
     ] + select({
-        "@platforms//os:linux": [
-            ":incompatible-os",
-        ],
         "@platforms//os:macos": [
             ":xcode-autobuilder",
         ],
+        "//conditions:default": [
+            ":incompatible-os",
+        ],
```

Comment:
shouldn't this be something like
```bazel
pkg_filegroup(
    name = "extractor-pack-arch",
    srcs = select({
        "@platforms//os:windows": [],
        "//conditions:default": [
            ":extractor",
            ":swift-test-sdk-arch",
        ],
    }) + select({
        "@platforms//os:macos": [
            ":xcode-autobuilder",
        ],
        "//conditions:default": [
            ":incompatible-os",
        ],
    }),
    visibility = ["//visibility:public"],
)
```
or should this be postponed to a follow-up PR where the whole `extractor-pack` can be built on Windows (including a Windows version of `extractor-pack-generic`, probably with just an `autobuild.cmd`?

Reply:
Yes, I was planning on doing this in a follow-up PR, but see my comment to Geoff below: I'm now leaning towards adding all the commits here instead.